.github/workflows/python.yml

```diff
@@ -209,3 +209,99 @@ jobs:
           make travis-test-postgres
         env:
           TEST_GROUP: ${{ matrix.instance }}
+
+  test-py3-snuba:
+    name: 'py3 snuba [ignore fails]'
+    runs-on: ubuntu-16.04
+    continue-on-error: true
+    strategy:
+      matrix:
+        instance: [0, 1]
+
+    env:
+      SENTRY_PYTHON3: 1
+      PIP_DISABLE_PIP_VERSION_CHECK: on
+
+      USE_SNUBA=: 1
```

Comment:
```suggestion
      USE_SNUBA: 1
```

Reply:
By the way, getting this right only changes `sentry.tsdb.inmemory.InMemoryTSDB` -> `sentry.tsdb.redissnuba.RedisSnubaTSDB`. The search and eventstream backend are still snuba.

Reply:
I'd be in favor of removing `USE_SNUBA` to reduce confusion and just having `RedisSnubaTSDB` as default, since snuba devservice is basically essential to almost all tests in my experience.

Reply:
Never mind, its for a fast testing path.